client/vite.config.ts

```diff
@@ -1,5 +1,5 @@
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+import { defineConfig } from "vite";
```

Comment:
What are folks' thoughts on adding a config for formatting code (i.e., Prettier)? It's minimal in this PR but could be nice for preventing unnecessary additions like this file.